Content.Server/GameTicking/GameTicker.LobbyBackground.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.IO;
+using System.Linq;
+using Content.Shared.Audio;
+using Robust.Shared.ContentPack;
+using Robust.Shared.Random;
+using Robust.Shared.Utility;
+using Robust.Shared.ViewVariables;
+
+
+namespace Content.Server.GameTicking;
+
+public sealed partial class GameTicker
+{
+    [Dependency] private readonly IResourceManager _resourceManager = default!;
+
+    [ViewVariables]
+    public string? LobbyBackground { get; private set; }
+
+    [ViewVariables]
+    private List<ResourcePath>? _lobbyBackgrounds;
+
+    private static readonly string[] WhitelistedBackgroundExtensions = new string[] {"png", "jpg", "jpeg"};
+
+    private const string LobbyScreenPath = "/Server Info/LobbyScreens";
```

Comment:
`Server Info` is a bad path for this, it should be in `Textures`

Reply:
Yep good call, move to textures. I thought it made more sense to be in server info but that was at 1am :P